docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3'
+
+services:
+  #find a way to tear this down after the init scripts has worked
+  setup-datastore:
+    build: ./scripts/setup-datastore/
+    depends_on:
+      - datastore
+    environment:
+      MEILISEARCH_HOST: http://meilisearch:7700
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/key/creds.json
+      GCLOUD_PROJECT: pname
+      DATASTORE_EMULATOR_HOST: datastore:8000
+      DATASTORE_PROJECT_ID: pname
+
+  #find a way to tear this down after the init scripts has worked
+  setup-meilisearch:
+    build: ./tools/meilisearch-index-generator/
+    depends_on:
+      - setup-datastore
+      - meilisearch
+    environment:
+      MEILISEARCH_HOST: http://meilisearch:7700
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/key/creds.json
+      GCLOUD_PROJECT: pname
+      DATASTORE_EMULATOR_HOST: datastore:8000
+      DATASTORE_PROJECT_ID: pname
+
+  meilisearch:
+    image: getmeili/meilisearch
+    volumes:
+      - ./data.ms:/data.ms
+    ports:
+      - 7700:7700
+
+  datastore:
+    image: google/cloud-sdk
```

Comment:
The google/cloud-sdk image is >~ 1GB ? Can't the alpine or slim version be used instead ?

Reply:
I will create a custom image based on  this file https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/master/emulators/Dockerfile